.changes/1923-general-ui-fixes.md

```diff
@@ -0,0 +1 @@
+[fixes] : General UI color issues fixes
```

Comment:
Let's keep messages end-user relevant (or out of this file). "General XYZ" doesn't tell the user anything and thus is meaningless:
```suggestion
- Switching to black background and bolder blue colors for highlights for better contrast
- Fix to make sure the description of the visibility items is legible 
```

Reply:
Noted! Thanks for informations :)